doc/conf.py

```diff
@@ -302,7 +302,8 @@
 sphinx_gallery_conf = {
     'doc_module': 'nilearn',
     'backreferences_dir': os.path.join('modules', 'generated'),
-    'reference_url': {'nilearn': None}
+    'reference_url': {'nilearn': None},
+    'filename_pattern': '/plot_compare_decomposition.py'
```

Comment:
That should not be there. Why did you have to add it?

Reply:
Added to make the make html only render the file I was, otherwise takes forever on my computer. Was not supposed to be pushed, just forgot to checkout the change. Will fix now!